Data.json

```diff
@@ -253,5 +253,10 @@
     "name": "Anjali Prasad",
     "username": "anjali112-bit",
     "avatar": "https://github.com/anjali112-bit.png"
+  },
+  {
+    "name": "Vaibhav Lonari",
+    "username": "gauravlonari",
+    "avatar": "https://github.com/gauravlonari.png"
```

Comment:
Your using two names here Vaibhav Lonari & Gaurav Lonari. Please make sure to use one.